app/views/shared/proxy_rules/_table.html.slim

```diff
@@ -33,7 +33,7 @@ table#mapping_rules.data
         td.delta
           = rule.delta
         td.metric
-          = rule.metric_system_name
+          = rule.metric.friendly_name
```

Comment:
Is this "rule" always the decorator? if so, then I think we should do there `delegate :system_name, :friendly_name, to: :metric, prefix: true` and here `rule.metric_friendly_name`

So the less that we spread the "talking to neighbors", the better 😄

Reply:
I think it may not be always decorated. What you suggested was the first thing I tried and it did not work. I think `metric_system_name` was actually from https://github.com/3scale/porta/blob/179cd1fd95ff5ecee03618311053f2f81fb698cf/app/models/proxy_rule.rb#L105.